source/_docs/z-wave/controllers.markdown

```diff
@@ -63,6 +63,10 @@ The alternative to a stick is a hub that supports Z-Wave. Home Assistant support
 
 ## Controller Notes
 
+### 800 Series Controllers
+
+Some features are not supported with 800 series controllers yet. You can find the full list of supported and unsupported features with 800 seires controllers [here](https://github.com/zwave-js/node-zwave-js/issues/5257).
```

Comment:
If there is information in the GitHub issue, we should list that here and not link to issues.